Tasks/AzureFileCopyV3/task.json

```diff
@@ -226,6 +226,13 @@
             "defaultValue": "",
             "groupName": "output",
             "helpMarkDown": "Provide a name for the variable for the storage container SAS Token used to access the files copied to with this task.  Valid only when the selected destination is Azure Blob."
+        },
+        {
+            "name": "sasTokenTimeOutInMinutes",
+            "type": "string",
+            "label": "SAS Token Expiration Period In Minutes",
+            "required": false,
+            "helpMarkDown": "Enter the time in minutes after which SAS token expire."
```

Comment:
Enter the time in minutes after which SAS token will expire.

Reply:
It has been done